Cpp/odin-views/app_secure.cpp

```diff
@@ -39,7 +39,8 @@ namespace {
             fostlib::pg::connection &cnx,
             fostlib::json jwt_header,
             fostlib::json jwt_body) {
-        const auto app_id = fostlib::coerce<fostlib::string>(jwt_body["iss"]);
+        auto const jwt_iss = fostlib::coerce<fostlib::string>(jwt_body["iss"]);
+        auto const app_id = jwt_iss.substr(odin::c_app_namespace.value().length());
```

Comment:
This code pre-supposes that the prefix exists on the `iss` field. Probably it should check

Reply:
If the prefix doesn't exist on the `iss` field, what happens? throw exception?